$OpenBSD: patch-src_enblend_cc,v 1.3 2010/07/11 20:09:00 sthen Exp $

we don't have fenv.h/fesetround(), but we can set rounding behaviour
this way instead. probably a noop anyway; FP_RN is the default.

--- src/enblend.cc.orig	Sun Dec 20 15:32:28 2009
+++ src/enblend.cc	Wed Feb 10 08:38:35 2010
@@ -50,8 +50,14 @@
 extern "C" char *optarg;
 extern "C" int optind;
 
+#ifdef __OpenBSD__
+#include <ieeefp.h>
+#define fesetround(x) fpsetround(x)
+#define FE_TONEAREST FP_RN
+#else
 #ifndef _MSC_VER
 #include <fenv.h>
+#endif
 #endif
 
 #include <signal.h>
